app/livechat/client/views/app/livechatReadOnly.js

```diff
@@ -12,7 +12,8 @@ import { inquiryDataStream } from '../../lib/stream/inquiry';
 Template.livechatReadOnly.helpers({
 	inquiryOpen() {
 		const inquiry = Template.instance().inquiry.get();
-		return inquiry && inquiry.status === 'queued';
+		const room = Template.instance().room.get();
+		return (inquiry && inquiry.status === 'queued') || !room.servedBy;
```

Comment:
Hey Murtaza, I know this is an old PR, but, would you mind explaining me this change? 👀

Reply:
Hey, @KevLehman This change was needed to ensure that we show the `Take It` button on the chat if there are no agents assigned to a chat yet. 
Before this change, we were only showing that button if the "Manual Selection" algorithm was on. However, we ran into a situation (on servers running other algorithms than manual) where LiveChat managers were using the "Current Chats" panel to send messages to unassigned chats - this resulted in chats that were actually answered but not assigned to any agents - since the manually answered those chats and didn't get the routing algorithm involved.

Hence we added a new rule here where we decided that if any manager wants to answer an unassigned chat, then they'd first have to "Take it" ..... Hence I added this new condition here - `!room.servedBy`.

Let me know if you need any further help with this :smile: